modules/oxide/src/less/theme/globals/global-variables.less

```diff
@@ -62,7 +62,7 @@
 @z-index-menu: 1150;
 @z-index-fullscreen: 1200;
 @z-index-sink: 1300;
```

Comment:
It becomes hard to tell what order these are displayed with the number being the same thing without inspecting the dom. Maybe change this to 1250. It should be lower than the sink.

Reply:
The throbber needs to be equal or less than the sink for the notifications to be on top... or did you mean make the throbber 1250?

Reply:
Ahh, yes thobber just making the values show in what order things are displayed would be nice.

Reply:
This feels like old c64 basic.

Reply:
CSS tries to hide the composition rules, but it's all too easy to get stuck in a weird dance with it